Simulation/controllers/server_controller/server_controller.py

```diff
@@ -112,7 +117,8 @@ def send_message(message):
 
 # Determine whether 2 bots have swapped in the past
 def have_swapped(b1, b2):
-    return b2 in b1.swapped_with or b1 in b2.swapped_with
```

Comment:
Snap niet helemaal wat je met deque bedoelt, zou je dit kunnen toelichten?

Reply:
deque is essentially een stack, maar dan FIFO ipv LIFO. Dan worden swaps tussen twee bots na een x aantal andere swaps weer toegestaan